dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/ContentletAPI.java

```diff
@@ -142,6 +144,17 @@
 	 */
 	public List<Contentlet> findContentletsByFolder(Folder parentFolder, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;
 
+	/**
+	 * Gets a list of Contentlets using a specific Template
+	 * @param template
+	 * @param user to check for permissions
+	 * @param respectFrontendRoles
+	 * @return
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	public List<Contentlet> findPagesByTemplate(Template template, User user, boolean respectFrontendRoles) throws  DotDataException, DotSecurityException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 152 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)